app/models/provider_user.rb

```diff
@@ -5,6 +5,9 @@ class ProviderUser < ApplicationRecord
   has_many :providers, through: :provider_permissions
   has_many :notes, dependent: :destroy
   has_many :pool_invites, class_name: 'Pool::Invite', foreign_key: 'invited_by_id'
+  has_one :up_to_date_find_a_candidate_all_filter, -> { find_candidates_all.order('updated_at desc') }, class_name: 'ProviderUserFilter'
```

Comment:
```suggestion
  has_one :find_a_candidate_all_filter, -> { find_candidates_all.order('updated_at desc') }, class_name: 'ProviderUserFilter'
```

How do you feel about removing the `up_to_date` part of these methods and scopes?